addons/addons.json

```diff
@@ -142,6 +142,7 @@
   "auto-follow-topics",
   "featured-dangos",
   "remember-collapsed-categories",
+  "yippee",
```

Comment:
Addon IDs are important for developers. Can you change it to something clearer than `yippee` such as `accessibility-tools`?